tests/query_context_tests.py

```diff
@@ -0,0 +1,94 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would it make sense to place this on /tests/charts/query_context_tests ?

Reply:
These tests are for `query_context.py` and `query_object.py` in `superset/common`, and in theory are not directly related to `tests/charts`.

Reply:
ok